src/test/ui/consts/const_let_eq_float.rs

```diff
@@ -1,6 +1,6 @@
 // compile-pass
 
-#![feature(const_let, const_fn)]
+#![feature(const_fn)]
```

Comment:
And now this should be a `compile-fail` test for stable or at least duplicate the file without any feature gates and then it should have errors...

Reply:
There's little point int making this a compile-fail test as that would just be another test for floats being forbidden in min const fn. We already have quite a bunch of those

Reply:
Assign-ops need a variable to assign to, and that variable can't possibly contain anything floating point related or you'd get an error. Assign-ops to floats aren't specially forbidden, so there's nothing new to test here.

Reply:
Ah that makes sense.